migration/sql-files/027-areas-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX area_path_btree_index ON areas USING BTREE (path);
```

Comment:
Actually No.
The `gist` index does it all. Removed in https://github.com/almighty/almighty-core/pull/796/commits/33731c8e169261c29803d0d11e00350d1cfdd312